SS_readdata_330.tpl

```diff
@@ -260,22 +262,25 @@
         {
           N_catchfleets(0)++;  //  overall N
           N_catchfleets(p)++;  //  count by area
-          fish_fleet_area(0,N_catchfleets(0))=f;  //  to find the original fleet index
-          fish_fleet_area(p,N_catchfleets(p))=f;  //  to find the original fleet index
+          fish_fleet_area(0,N_catchfleets(0))=f;  //  to find the "f" index for a catchfleet when not within an area loop
```

Comment:
style: missing space?

Reply:
readdata and readcontrol were not automatically styled